packages/babel-minify/src/index.js

```diff
@@ -5,26 +5,50 @@ module.exports = function babelMinify(
   input,
   // Minify options passed to minifyPreset
   // defaults are handled in preset
-  options = {},
+  minifyOpts = {},
   // overrides and other options
   {
     minified = true,
     inputSourceMap = null,
     sourceMaps = false,
+    comments = /^\**!|@preserve|@license|@cc_on/,
 
     // to override the default babelCore used
     babel = babelCore,
 
     // to override the default minify preset used
-    minifyPreset = babelPresetMinify
+    minifyPreset = babelPresetMinify,
+
+    filename,
+    filenameRelative
   } = {}
 ) {
   return babel.transform(input, {
+    minified,
     babelrc: false,
-    presets: [[minifyPreset, options]],
-    comments: false,
-    inputSourceMap,
+    ast: false,
+
+    presets: [[minifyPreset, minifyOpts]],
+
     sourceMaps,
-    minified
+    inputSourceMap,
+
+    shouldPrintComment(contents) {
+      return shouldPrintComment(contents, comments);
```

Comment:
I believe it's a [babel-generator option](https://github.com/babel/babel/blob/b3372a572db511db44fd151a69375dbb4b8d0573/packages/babel-generator/src/index.js#L43).